app/components/UserSearchResult.tsx

```diff
@@ -0,0 +1,75 @@
+import { USER_SEARCH_PAGE } from "~/utils/urls";
+
+export function UserSearchResult({
+  discordUsername,
+  discordId,
+  twitterHandle,
+}: {
+  discordUsername: string;
+  discordId: number;
+  twitterHandle?: string;
+}) {
+  const userPageHyperlink = `${USER_SEARCH_PAGE}/${discordId}`;
+
+  // Retrieves the string content before the '#' character in the full Discord username
+  let discordAvatarAltText: string | undefined;
+  const regexExtractDiscordNameWithoutIdentifier = new RegExp("^(.+?)#");
+  const regexArray =
+    regexExtractDiscordNameWithoutIdentifier.exec(discordUsername);
+  if (regexArray !== null) {
+    discordAvatarAltText = regexArray[0];
+  }
```

Comment:
No need for alt text for the ava. The name itself as displayed on the screen describes it.

Reply:
Hmm, VSCode is telling me that the `<img />` tag needs an alt text of some kind - though I think that's from a plugin I use , jsx-a11y:

![image](https://user-images.githubusercontent.com/15804376/199597437-ee678dec-9fea-469d-b7a3-ae66fae690b6.png)

I'll still remove it though as you mentioned

Reply:
```
C:\Users\ElementUser\_dev\Github\sendou.ink\app\routes\u.tsx
  61:11  warning  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
```

IMO, we should include alt text after all, one of the linters is giving an error. And also I was mistaken, looks like jsx-a11y is actually something used by this project & isn't something different from my computer (the pipeline also outputs this warning)

Reply:
For now, I changed the alt to just use the Discord Username (so no regex parsing or anything)

Reply:
> or an empty string for decorative images

So in this case just doing alt="" would be correct I believe :) There is not really anything useful we can explain to screen reader users assuming we also have the user name next to it.

Reply:
Fixed